### ISSUE TITLE
Dynamic Typing changes bit int fields

### ISSUE THREAD

**Issue by lebedevat** [mholt/PapaParse#692]
For `dynamicTyping: true` parser changes big int fields. For example, file below without dynamicTyping
```
["208310224003697457", "1", "0"]
["208310224003697685", "1", "0"]
```
with dynamicTyping
```
[208310224003697470, 1, 0]
[208310224003697700, 1, 0]
```
[big_int_example.txt](https://github.com/mholt/PapaParse/files/3394747/big_int_example.txt)

**Comment by pokoli** [comment]
I think big integer fields are not supported in dynamic typing, so a pull request adding support for it will very welcome.

**Comment by haxxxton** [comment]
literally just came across this same problem, will investigate a pull request next week if its not already done

**Comment by pokoli** [comment]
Fixed in #694
